--- caches/icache.p	2014-07-16 15:35:56.387982488 -0400
+++ caches/icache.p.new	2014-07-16 15:34:48.234986425 -0400
@@ -1,11 +1,11 @@
 set term pngcairo dashed
-set title "Instruction Cache"
+set title "bsort100 Instruction Cache"
 set output "icache.png"
 set xlabel "Program Point"
 set ylabel "UCBs Shared with Subsequent Program Point"
 set xtics 0,1
-set xrange []
-set yrange []
+set xrange [0:16]
+set yrange [120:190]
 # Minimum Line Style
 set linestyle 1 lt 1 lc rgb "black"
 # Maximum Line Style
