--- caches/dcache.p.orig	2014-07-16 17:24:09.769575204 -0400
+++ caches/dcache.p	2014-07-16 17:25:26.924570747 -0400
@@ -1,11 +1,11 @@
 set term pngcairo dashed
-set title "Data Cache"
-set output "dcache.png"
+set title "recursion Data Cache"
+set output "recursion-dcache.png"
 set xlabel "Program Point"
 set ylabel "UCBs Shared with Subsequent Program Point"
 set xtics 0,1
-set xrange []
-set yrange []
+set xrange [0:11]
+set yrange [0:30]
 # Minimum Line Style
 set linestyle 1 lt 1 lc rgb "black"
 # Maximum Line Style
