--- caches/icache-eps.p.orig	2014-07-16 17:23:59.915575774 -0400
+++ caches/icache-eps.p	2014-07-16 17:28:52.113558893 -0400
@@ -1,11 +1,11 @@
 set term postscript eps color
-set title "Instruction Cache"
-set output "icache.eps"
+set title "recursion Instruction Cache"
+set output "recursion-icache.eps"
 set xlabel "Program Point"
 set ylabel "UCBs Shared with Subsequent Program Point"
 set xtics 0,1
-set xrange []
-set yrange []
+set xrange [0:11]
+set yrange [120:165]
 # Minimum Line Style
 set linestyle 1 lt 1 lc rgb "black"
 # Maximum Line Style
