--- caches/icache.p.orig	2014-10-11 14:53:46.798554330 -0400
+++ caches/icache.p	2014-10-11 14:57:39.422560560 -0400
@@ -1,11 +1,11 @@
 set term pngcairo dashed
-set title "Instruction Cache"
-set output "icache.png"
+set title "fibcall Instruction Cache"
+set output "fibcall-icache.png"
 set xlabel "Program Point"
 set ylabel "UCBs Shared with Subsequent Program Point"
 set xtics 0,1
-set xrange []
-set yrange []
+set xrange [0:12]
+set yrange [130:145]
 # Minimum Line Style
 set linestyle 1 lt 1 lc rgb "black"
 # Maximum Line Style
